iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/manager/load/cache/node/DataNodeHeartbeatCache.java

```diff
@@ -38,10 +38,6 @@ public DataNodeHeartbeatCache(int dataNodeId) {
 
   @Override
   public synchronized void updateCurrentStatistics() {
-    // The Removing status can not be updated
```

Comment:
Why remove this line?Then the remove status maybe updated by Running or Readonly or Unknown?